icu4j/main/classes/core/src/com/ibm/icu/text/ConstrainedFieldPosition.java

```diff
@@ -193,8 +193,10 @@ public void constrainClass(Class<?> classConstraint) {
      *
      * @param field The field to fix when iterating.
      * @param fieldValue The field value to fix when iterating.
-     * @internal ICU 64 Technical Preview
+     * @internal ICU 64 technology preview
+     * @deprecated This API is for technology preview and might be changed or removed in a future release.
```

Comment:
API tool requires `@deprecated` is used along with `@internal`. The description of `@internal` should contain the phrase 'technology preview', that is recognized by the tool. Although, we may revisit this to change the restriction after 64.